driver.md

```diff
@@ -353,7 +354,12 @@ State Manipulation
     syntax EthereumCommand ::= "clear"
  // ----------------------------------
     rule <k> clear => clearTX ~> clearBLOCK ~> clearNETWORK ... </k>
-         <analysis> _ => .Map </analysis>
+         <analysis> _ => "coverage" |-> .Set </analysis>
+         <mode> EXECMODE </mode>
+      requires EXECMODE in (SetItem(NORMAL) SetItem(VMTESTS))
+
+    rule <k> clear => clearTX ~> clearBLOCK ~> clearNETWORK ... </k>
+         <mode> COVERAGE </mode>
```

Comment:
um, why do we need this?

Reply:
In `normal` or in `vmtest` mode the `analysis` cell has its contents removed at the `clear` computation so I needed to remove this behaviour when the mode is `coverage`.